name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/JTDirektbankPDFExtractor.java

```diff
@@ -0,0 +1,153 @@
+package name.abuchen.portfolio.datatransfer.pdf;
+
+import java.util.Map;
+import java.util.function.BiConsumer;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import name.abuchen.portfolio.datatransfer.DocumentContext;
+import name.abuchen.portfolio.datatransfer.pdf.PDFParser.Block;
+import name.abuchen.portfolio.datatransfer.pdf.PDFParser.DocumentType;
+import name.abuchen.portfolio.datatransfer.pdf.PDFParser.ParsedData;
+import name.abuchen.portfolio.datatransfer.pdf.PDFParser.Transaction;
+import name.abuchen.portfolio.model.AccountTransaction;
+import name.abuchen.portfolio.model.Client;
+
+@SuppressWarnings("nls")
+public class JTDirektbankPDFExtractor extends AbstractPDFExtractor
+{
+
+    private static final String DEPOSIT_2023 = "^(?<date>\\d+.\\d+.) (\\d+.\\d+.) (.*gutschr\\.?)(\\s*)(?<amount>[\\d\\s,.]*) [H]";
+    private static final String REMOVAL_2023 = "^(?<date>\\d+.\\d+.) (\\d+.\\d+.) (Überweisungsauftrag)(\\s*)(?<amount>[\\d\\s,.]*) [S]";
+    private static final String INTEREST_2023 = "^(?<date>\\d+.\\d+.) (\\d+.\\d+.) (Abschluss lt\\. Anlage \\d) (?<amount>[\\d\\s,.]*?) (.*)";
+
+    private static final String CONTEXT_KEY_YEAR = "year";
+    private static final String CONTEXT_KEY_CURRENCY = "currency";
+    private static final String CONTEXT_KEY_TRANSACTIONS_HAVE_FULL_DATE = "transactions_full_date";
+    private static final String CONTEXT_VALUE_TRUE = "true";
```

Comment:
Hello @ZfT2
thx... is it possible for you to adapt the [regular expression to our standard](https://github.com/portfolio-performance/portfolio/blob/master/CONTRIBUTING.md#regular-expressions)?

```suggestion
    private static final String DEPOSIT_2023 = "^(?<date>[\\d]{2}\\.[\\d]{2}\\.) [\\d]{2}\\.[\\d]{2}\\. (.*gutschr\\.?)[\\s]{1,}(?<amount>[\\.,\\d]+) [H]";
    private static final String REMOVAL_2023 = "^(?<date>[\\d]{2}\\.[\\d]{2}\\.) [\\d]{2}\\.[\\d]{2}\\. (.berweisungsauftrag)[\\s]{1,}(?<amount>[\\.,\\d]+) [S]";
    private static final String INTEREST_2023 = "^(?<date>[\\d]{2}\\.[\\d]{2}\\.) [\\d]{2}\\.[\\d]{2}\\. (Abschluss lt\\. Anlage \\d)[\\s]{1,}(?<amount>[\\.,\\d]+).*";

    private static final String CONTEXT_KEY_YEAR = "year";
    private static final String CONTEXT_KEY_CURRENCY = "currency";
    private static final String CONTEXT_KEY_TRANSACTIONS_HAVE_FULL_DATE = "transactions_full_date";
    private static final String CONTEXT_VALUE_TRUE = "true";
```